packages/api/src/db/block.ts

```diff
@@ -85,6 +85,9 @@ export interface EmbedBlock {
   title?: string
   width?: number
   height?: number
+  styleHeight?: string
+  styleWidth?: string
+  ratio?: boolean
```

Comment:
maybe a more explaining name like `useRatio` and i'm not sure if a boolean should be optional. I'd suggest a default in this case maybe true or wathever it was before.

Reply:
yes indeed the default of this is `true`, currently set as initial state

Reply:
ratio is dropped